src/NuGet.Clients/PackageManagement.PowerShellCmdlets/Cmdlets/InstallPackageCommand.cs

```diff
@@ -99,6 +100,15 @@ protected override void ProcessRecordCore()
             {
                 await InstallPackageByIdAsync(Project, Id, ResolutionContext, this, WhatIf.IsPresent);
             }
+            catch (FatalProtocolException ex)
+            {
+                // Wrap FatalProtocolException coming from the server with a user friendly message
+                var error = String.Format(CultureInfo.CurrentUICulture, Strings.Exception_PackageNotFound, Id, Source);
+                Log(MessageLevel.Error, error);
+
+                // Additional information about the exception can be observed by using the -verbose switch with the install-package command
+                Log(MessageLevel.Debug, ExceptionUtilities.DisplayMessage(ex));
```

Comment:
How about this - 
![image](https://cloud.githubusercontent.com/assets/10507120/16931629/c39d3d54-4cf6-11e6-8c95-5efbe480331a.png)